### DISCUSSION TITLE
ODataQueryOptions and ODataQueryOptions<T> parameter for HttpGet stops generation of OpenAPI.json file

### DISCUSSION THREAD

**Discussion by AndrewZenith** [microsoft/OpenAPI.NET#1937] [General]
using Microsoft.AspNetCore.OData.Query;

[HttpGet("languages")]
public IQueryable<Language> Get(ODataQueryOptions options)
or
[HttpGet("languages")]
public IQueryable<Language> Get(ODataQueryOptions<Language> options)

Both of these cause the OpenApi file generation to fail.

Is there any way of getting explicit Odata parameter options in the OpenApi file to be consumed by a UI?

**Comment by baywet** [comment]
This issue sounds like it originates in the ASP.NET core implementation of OpenAPI (Microsoft.AspNetCore.OpenApi). Please reach out on [the aspnet repository](https://github.com/dotnet/aspnetcore) instead.

Alternatively, if you're using Swashbuckle in your application, you may reach out [on the swashbuckle repository](https://github.com/domaindrivendev/Swashbuckle.AspNetCore).
